tasks/electron-react-ts/src/App.tsx

```diff
@@ -0,0 +1,80 @@
+import { DocumentIDValue } from '@dittolive/ditto'
+import { useMutations, usePendingCursorOperation } from '@dittolive/react-ditto'
+import React, { useEffect, useState } from 'react'
+import './App.css'
+
+const COLLECTION = 'tasks'
+
+type Task = { _id?: DocumentIDValue; body: string; isCompleted: boolean }
+
+const App = () => {
+  const [text, setText] = useState('')
+  const { ditto, documents: tasks } = usePendingCursorOperation<Task>({
+    collection: COLLECTION,
+  })
+  const { insert, removeByID, updateByID } = useMutations<Task>({
+    collection: COLLECTION,
+  })
+
+  useEffect(() => {
+    ditto?.startSync()
```

Comment:
You'll probably want to stop syncing here once the component is unmounted.

Reply:
That's right, thanks for pointing that out!